script/elm_format.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python
```

Comment:
One of the things that I dislike about using buck that it introduces so many different languages. I'd much rather do this in haskell than in python (maybe nix-script-haskell?).

Reply:
They recommend using Python scripts because there’s an easy Python bootstrap path. We can certainly move these to a different language if we’re willing to add the right compiler tool chains! I just haven’t been so far. This particular script is generally useful, and we’ll probably end up rewriting it in something else.